sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/statsbeat/_customer_sdkstats.py

```diff
@@ -212,28 +209,28 @@ def _get_drop_reason(self, drop_code: DropCodeType, exception_message: Optional[
             return categorize_status_code(drop_code)
 
         if drop_code == DropCode.CLIENT_EXCEPTION:
-            return exception_message if exception_message else "unknown_exception"
+            return exception_message if exception_message else _CLIENT_EXCEPTION
 
         drop_code_reasons = {
-            DropCode.CLIENT_READONLY: "readonly_mode",
-            DropCode.CLIENT_STORAGE_DISABLED: "local storage is disabled",
-            DropCode.CLIENT_PERSISTENCE_CAPACITY: "persistence_full",
+            DropCode.CLIENT_READONLY: "Readonly mode",
```

Comment:
Let's try to align with node. @JacksonWeber  What are the reasons that node exactly for these cases?